recipe/bld.bat

```diff
@@ -9,7 +9,6 @@ cmake ^
     -G "Ninja" ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
     -DCMAKE_BUILD_TYPE=Release ^
-    -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Comment:
Is this now being taken care of upstream?

Reply:
Yes, the CMake build system of sdl12-compat has proper support for marking Windows symbols as visible.